src/lms-ds-loader/tests/test_csv_to_sql.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import pandas as pd
+import sqlalchemy as sal
+
+
+from lms_ds_loader.csv_to_sql import CsvToSql
+
+
+class Test_when_reading_csv_and_loading_into_a_database:
+    class Test_given_valid_arguments:
+        def test_then_load_file_into_specified_table_using_pandas(self, mocker):
+            connection_string = "23456789"
+            file = "a/b/c.csv"
+            table = "c"
+
+            # Arrange

```

Comment:
I often put these `# Arrange`, `# Act`, `# Assert` comments into tests to help the reader better understand what is going one. It stems from my frustration at looking at long tests and not even being able to find the "action" amidst all of the noise - the the thing being tested.

Reply:
Looks good